tests/test_controller.py

```diff
@@ -236,9 +377,15 @@ def setup(self):
             protocol=remote_collection.protocol
         )
 
+        # Create two works to use in tests.
         self.work1 = self._work(with_open_access_download=True)
         self.work2 = self._work(with_open_access_download=True)
 
+        # Clear the cached OPDS entries for one of these works -- this
+        # verifies that OPDS entries are created when needed.
+        self.work1.verbose_opds_entry = None
+        self.work1.simple_opds_entry = None
+
```

Comment:
This guarantees we run through the code path that caused the "wrong order" bug fixed here: https://github.com/NYPL-Simplified/metadata_wrangler/pull/199/files#r314781382